catalog/core/catalog-core-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -27,15 +27,20 @@
 
     <bean id="catalogMigratable"
           class="org.codice.ddf.catalog.migratable.impl.CatalogMigratableImpl">
+        <argument ref="catalogDescribable"/>
         <argument ref="catalogFramework"/>
         <argument ref="filterBuilder"/>
         <argument ref="catalogFileWriter"/>
         <argument ref="catalogMigratableConfig"/>
+    </bean>
+
+    <bean id="catalogDescribable"
+          class="org.codice.ddf.migration.DescribableBean">
         <argument value="1.0"/>
-        <argument value="org.codice.ddf.catalog"/>
+        <argument value="ddf.catalog"/>
```

Comment:
I'm going to throw a curve ball here and ask this: should we name this `ddf.metacards` and rename the class, beans, etc., `Metacards` or `metacards` something? The only thing we are and will be migrating in this package are the metacards. Other things stored in the catalog will have their own migratables (e.g., `WorkspacesMigratable`, `MetacardVersionsMigratable`, `RegistryMigratable`, etc.) in the future.